src/mumble/Settings.cpp

```diff
@@ -401,7 +401,16 @@ Settings::Settings() {
 	bJackStartServer  = false;
 	bJackAutoConnect  = true;
 
-	echoOption = EchoCancelOptionID::DISABLED;
+#ifdef Q_OS_MAC
```

Comment:
```c
#ifdef Q_OS_MACOS
```

From https://doc.qt.io/qt-5/qtglobal.html#Q_OS_MAC:
> Deprecated synonym for **Q_OS_DARWIN**. Do not use.

Reply:
Interestingly we are using `Q_OS_MAC` in a lot of places in this file. I will add a commit that fixes that as well

Reply:
Not only in this file, unfortunately.

We could simply perform a search & replace to fix all instances.

Reply:
That'd probably be worth it in order to avoid copy&paste errors as this one.
Do you want to do it or shall I?